### PR TITLE
Add ironic image version in kustomization.yaml example

### LINKED ISSUE #388: Pin ironic image for the quick-start
After https://github.com/metal3-io/ironic-image/pull/479 was merged, the default changed to not use inspector. This breaks the [quick-start guide](https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/src/quick-start.md?plain=1) since it is using BareMetalOperator v0.5.0 that assumes inspector is used.
We should pin the ironic-image to v24.0.0 to fix this and avoid similar issues going forward.

Add the following to the [ironic kustomization](https://github.com/metal3-io/metal3-docs/blob/072349dac347b9ae39c94f1aaf2ec750db642a0a/docs/user-guide/src/quick-start.md?plain=1#L415) in the quick-start:

```yaml
images:
- name: quay.io/metal3-io/ironic
  newTag: v24.0.0
```

### PR DESCRIPTION
**Pull request by lekaf974**
Fix #388

### CODE REVIEW

File: docs/user-guide/src/quick-start.md [hunk 1]
```diff
@@ -426,7 +426,9 @@ resources:
 components:
 - https://github.com/metal3-io/baremetal-operator/ironic-deployment/components/basic-auth?ref=v0.5.1
 - https://github.com/metal3-io/baremetal-operator/ironic-deployment/components/tls?ref=v0.5.1
-
+images:
+  - name: quay.io/metal3-io/ironic
```
Comment (tuminoid) [inline docs/user-guide/src/quick-start.md:430]:
```suggestion
- name: quay.io/metal3-io/ironic
```
Indent this the k8s way, ie. no indentation with lists.

**Review by tuminoid — CHANGES_REQUESTED** [review]

File: docs/user-guide/src/quick-start.md (hunk 1 above)
Comment (lekaf974) [reply docs/user-guide/src/quick-start.md]: done

**Comment by lekaf974** [comment]
@tuminoid changes done

**Review by tuminoid — COMMENTED** [review]
/lgtm

Thanks for the contribution!

**Review by tuminoid — APPROVED** [review]

**Pull request merged**